onnxruntime/core/providers/cpu/nn/conv_integer.cc

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/providers/cpu/nn/conv_integer.h"
```

Comment:
Disable warnings 4244 and 4267 for windows... this should resolve your windows build issue